[core] Clean error messages prefix (#2676)
mui-org/material-ui-x: docs/pages/api-docs/data-grid/data-grid-pro.md

| <span class="prop-name">isCellEditable</span> | <span class="prop-type">(params: GridCellParams) => boolean</span> |   | Callback fired when a cell is rendered, returns `true` if the cell is editable. |
| <span class="prop-name">isRowSelectable</span> | <span class="prop-type">(params: GridRowParams) => boolean</span> |   | Determines if a row can be selected. |
| <span class="prop-name">loading</span> | <span class="prop-type">boolean</span> |  false | If `true`, a  loading overlay is displayed.. |

> | <span class="prop-name">localeText</span> | <span class="prop-type">GridLocaleText</span> |   | Set of text labels used in the grid. You can find all the translation keys supported in [the source](https://github.com/mui-org/MUI-x/blob/HEAD/packages/grid/_modules_/grid/constants/localeTextConstants.ts) in the GitHub repository. |

Reviewer: ```suggestion
| <span class="prop-name">localeText</span> | <span class="prop-type">GridLocaleText</span> |   | Set of text labels used in the grid. You can find all the translation keys supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/constants/localeTextConstants.ts) in the GitHub repository. |
```

| Name | Type | Default | Description |
|:-----|:-----|:--------|:------------|

> | <span class="prop-name">Checkbox</span> | <span class="prop-type">elementType</span> | <span class="prop-type">Checkbox</span> | Checkbox component used in the grid for both header and cells. By default, it uses the MUI core Checkbox component. |

Reviewer: ```suggestion
| <span class="prop-name">Checkbox</span> | <span class="prop-type">elementType</span> | <span class="prop-type">Checkbox</span> | Checkbox component used in the grid for both header and cells. By default, it uses the MUI core Checkbox component. |
```

Author: What is the difference between the two lines here ?

Reviewer: It was the same oops. I wondered about the capitalization of the MUI Core.

Reviewer: If we approved this one, could we update the whole codebase? I assume that the source of truth for this one is in the TypeScript' descriptions.

Reviewer 2: > If we approved this one, could we update the whole codebase? I assume that the source of truth for this one is in the TypeScript' descriptions.

I think it's not worth it to waste time updating the manually authored API pages now. The documentation for this component can be automatically generated in #2465. BTW, the TypeScript definition doesn't have the "By default, it uses the MUI..." part.

Reviewer: > BTW, the TypeScript definition doesn't have the "By default, it uses the MUI..." part.

Oh nice, +1 for ignoring the api .md files and focusing on the TypeScript definition. It started to diverge on this instance with #1528.

- With a [global class name](/customization/components/#overriding-styles-with-global-class-names).
- With a theme and an [`overrides` property](/customization/globals/#css).

> If that's not sufficient, you can check the [implementation of the component style](https://github.com/mui-org/MUI-x/blob/master/packages/grid/_modules_/grid/components/containers/GridRootStyles.ts) for more detail.

Reviewer 2: 404

```suggestion
If that's not sufficient, you can check the [implementation of the component style](https://github.com/mui-org/material-ui-x/blob/next/packages/grid/_modules_/grid/components/containers/GridRootStyles.ts) for more detail.
```

Could you also point to next below?

https://github.com/mui-org/material-ui-x/blob/71cc0d3e6471cd6068a8fba555620953c553a377/docs/pages/api-docs/data-grid/data-grid.md#L219
